libcontainer/mount_linux.go

```diff
@@ -3,9 +3,19 @@ package libcontainer
 import (
 	"strconv"
 
+	"syscall"
+	"unsafe"
+
 	"golang.org/x/sys/unix"
 )
 
+const (
+	OPEN_TREE_CLONE         = 1
+	OPEN_TREE_CLOEXEC       = 02000000
+	MOVE_MOUNT_F_EMPTY_PATH = 0x00000004
+	AT_FDCWD                = -100
```

Comment:
Should these be defined in x/sys/unix? Please open a CL against it -- they are usually more than happy to accept such stuff.

Reply:
yeah, I wrote about it, in cover message, but definitely x/sys/unix is better candidate.

Reply:
@kolyshkin I've seen your https://go-review.googlesource.com/c/sys/+/363444 I assume this change was made w/o code generation e.g. by sys/unix/mkall.sh ?